Client-Side/shecodesprogramming/src/Components/HomePage/homepage.jsx

```diff
@@ -0,0 +1,109 @@
+                            <p>Programming involves solving problems using logical and algorithmic thinking. Programmers analyze problems, break them down into smaller components, and design algorithms or step-by-step instructions to solve them efficiently.</p>
+                        </div>
+                    </div>
+
+                    <div className="dsaimage">
+                        <div className="dsatext">
+                            <h2>What are Data Structures and Algorithms</h2>
+                            <p>Data structures are used to organize and store data in a program, while algorithms are the step-by-step procedures used to manipulate that data. Understanding different data structures and algorithms is crucial for efficient programming.</p>
+                        </div>
+                        <div>
+                            <img className="dsaImage" src="https://st.depositphotos.com/1654249/2526/i/600/depositphotos_25269101-stock-photo-3d-man-thinking-with-red.jpg" alt="" />
+                        </div>
+                    </div>
+
+                    <div className="displayoop">
+                        <div>
+                            <img className="oopimage" src="https://freepngimg.com/download/temp_png/72547-thinking-photography-question-mark-man-stock.jpeg" alt="" />   
+                        </div>
+                        <div className="ooptext">
+                            <h2>what is Object-Oriented Programming (OOP)</h2>
+                            <p>OOP is a programming paradigm that organizes code into reusable objects. It focuses on encapsulating data and functions into objects, allowing for modular and scalable code.</p>
+                        </div>
+                    </div>
+
+                    <div className="displaysldc">
+                        <div className="sldctext">
+                            <h2>Software Development Life Cycle (SDLC)</h2>
+                            <p>Programming is often part of the larger software development process. SDLC encompasses planning, requirements gathering, designing, coding, testing, deployment, and maintenance of software applications.</p>
+                        </div>
+                        <div>
+                            <img className="displaysldcimage" src="https://reachstudios.co.uk/wp-content/uploads/2021/09/SDLC-Banner-Image-300x300.jpg" alt="" />
+                        </div>
+                    </div>
+
+                    <div>
+                        <div>
+                            <div>
+                                <h2>Learn With Joy</h2>
+                            </div>
+                        </div>
+                        <div className="roundimages">
+                            <div>
+                                <img className="round1" src="https://uploads-ssl.webflow.com/620ce06fa12cad2d27fb30a8/6283b4f30d0b4a7295f5613f_green%20(1).png" alt="" />
```

Comment:
Please add big content in constant file and use that variable here.

Reply:
Okay